### ISSUE TITLE
HEAD is broken because of gazelle 0.9

### ISSUE THREAD

**Issue by jmhodges** [bazelbuild/rules_go#1396]
I'm not sure if y'all want to fix this or not, but I bumped rules_go without thinking I'd need to bump gazelle, and got these errors.

```
src/github.com/bazelbuild/bazel-gazelle/internal/merger/fix.go:93:28: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:657:16: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:677:19: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:681:19: too few values in struct initializer
# github.com/bazelbuild/bazel-gazelle/internal/rules
src/github.com/bazelbuild/bazel-gazelle/internal/rules/generator.go:183:16: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/rules/sort_labels.go:45:16: too few values in struct initializer
 and referenced by '//go/svc/howsmyssl-billing:go_default_library'
ERROR: Analysis of target '//go/svc/howsmyssl-billing:go_default_library' failed; build aborted: no such package '@com_github_jmhodges_clock//': no such package '@io_bazel_rules_go_repository_tools//': failed to build gazelle: # github.com/bazelbuild/bazel-gazelle/internal/merger
src/github.com/bazelbuild/bazel-gazelle/internal/merger/fix.go:93:28: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:657:16: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:677:19: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/merger/merger.go:681:19: too few values in struct initializer
# github.com/bazelbuild/bazel-gazelle/internal/rules
src/github.com/bazelbuild/bazel-gazelle/internal/rules/generator.go:183:16: too few values in struct initializer
src/github.com/bazelbuild/bazel-gazelle/internal/rules/sort_labels.go:45:16: too few values in struct initializer
````

**Comment by jmhodges** [comment]
That's with 

```
http_archive(
    name = "bazel_gazelle",
    url = "https://github.com/bazelbuild/bazel-gazelle/releases/download/0.9/bazel-gazelle-0.9.tar.gz",
    sha256 = "0103991d994db55b3b5d7b06336f8ae355739635e0c2379dea16b8213ea5a223",
)
```

**Comment by jayconrod** [comment]
Thanks for reporting. I've put out a PR that will fix the break.

Version 0.9 of Gazelle may be incompatible with the next version of rules_go for other reasons, which I'll note in the next release, but we should report it in a clear way instead of giving you build errors.

To fill in what happened here:

* Feb 20 - last commit before rules_go 0.10
* Feb 26 - `github.com/bazelbuild/buildtools/build` landed an API change which broke Gazelle. I updated Gazelle to work with the new API. I updated rules_go to point to the new Gazelle and the new buildtools.
* Mar 1 - buildtools promised several more breaking API changes. I vendored the packages I needed in Gazelle and removed the dependency.

So now, rules_go can point to the old version of buildtools to keep Gazelle 0.9 alive. Gazelle commits between Feb 26 and Mar 1 will break, but there weren't any releases in that period.
